hal/src/nRF52840/gsm0710muxer/platform.h

```diff
@@ -41,8 +43,16 @@ const auto taskPriority = OS_THREAD_PRIORITY_NETWORK;
 #define GSM0710_ASSERT(x) SPARK_ASSERT(x)
 
 // #define GSM0710_MODEM_STATUS_SEND_EMPTY_BREAK
-#define GSM0710_PUMP_INPUT_DATA 50
+// #define GSM0710_PUMP_INPUT_DATA 50
 // #define GSM0710_ENABLE_DEBUG_LOGGING
+#define GSM0710_SHARED_STREAM_EVENT_GROUP
+#define GSM0710_SHARED_STREAM_EVENT_BASE (HAL_USART_PVT_EVENT_END - 1)
```

Comment:
What does the value of `HAL_USART_PVT_EVENT_END` mean?

Reply:
The next flag bit we can use for our own purposes.

Reply:
Then it should use something like `__builtin_ffs`? It's currently set to 0x03 which is a combination of flag bits, not a single flag bit

Reply:
Yeah, let's make it a bit clearer. It does work _currently_ (https://github.com/particle-iot/gsm0710muxer/pull/5/files#diff-8d017d4d053da8f60b8c24e8c0f581a3R217) but agreed that it's easy to break.

Reply:
It does, it's just the formula that didn't make any sense to me

Reply:
Fixed.